unlock-app/src/components/interface/checkout/alpha/Checkout/Confirm.tsx

```diff
@@ -224,13 +228,51 @@ export function Confirm({
     }
   }
 
+  const onConfirmSuperfluid = async () => {
+    try {
+      if (payment.method !== 'superfluid') {
+        return
+      }
+      setIsConfirming(true)
+      const provider = selectProvider(config)
+      const web3Provider = new ethers.providers.Web3Provider(provider)
+      const sf = await Framework.create({
+        chainId: lock!.network,
+        provider: web3Provider,
+      })
+      const expiration = BigNumber.from(lock!.expirationDuration)
+      const flowRate = BigNumber.from(lock!.keyPrice)
+        .mul('1000000000000000000')
```

Comment:
Is this the number of decimals? Unfortunately not all ERC20 use the same value so we shouldn't hardcodecit